src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -666,11 +666,11 @@ They can be configured through the settings file 'winget settings'.</value>
     <value>No applicable update found.</value>
   </data>
   <data name="UpgradeCommandLongDescription" xml:space="preserve">
-    <value>Upgrades the selected package, either found by searching the installed packages list or directly from a manifest. By default, the query must case-insensitively match the id, name, or moniker of the package. Other fields can be used by passing their appropriate option.</value>
+    <value>Upgrades the selected package, either found by searching the installed packages list or directly from a manifest. By default, the query must case-insensitively match the id, name, or moniker of the package. Other fields can be used by passing their appropriate option.&#13;When no arguments are given, shows the packages with upgrades available</value>

```

Comment:
nit: the &#13; "\r" probably is not necessary since we don't have format characters in the resource file yet.

Reply:
Updated